ICAFIX/hcp_fix_multi_run

```diff
@@ -212,47 +221,30 @@ for fmri in $fmris ; do
 	log_Inform "tr: $tr"
 	log_Inform "processing FMRI file $fmri with highpass $hp"
 
-  #Demean volume, movement regressors, and data
+	#Demean movement regressors, volumes, and CIFTI data
+	demeanMovementRegressors Movement_Regressors.txt Movement_Regressors_demean.txt
+	MovementTXTMergeSTRING+="$(pwd)/Movement_Regressors_demean.txt "
+
 	${FSLDIR}/bin/fslmaths $fmri -Tmean ${fmri}_mean
 	${FSLDIR}/bin/fslmaths $fmri -sub ${fmri}_mean ${fmri}_demean
 
-	demeanMovementRegressors Movement_Regressors.txt Movement_Regressors_demean.txt
-	MovementTXTMergeSTRING=`echo "${MovementTXTMergeSTRING}$(pwd)/Movement_Regressors_demean.txt "`
+	${FSL_FIX_WBC} -cifti-reduce ${fmri}_Atlas.dtseries.nii MEAN ${fmri}_Atlas_mean.dscalar.nii
```

Comment:
needs to be ${fmriNoExt} like above instead of ${fmri}

Reply:
Actually, it doesn't at that point in the code, because starting at L205, $fmri is redefined to exclude any extension.  There is a comment at L206 to this effect.  I left the following code "as is" where possible, and only added $fmriNoExt where it was explicitly needed earlier in the code (to consolidate all the sub-shell calls to 'remove_ext').